shell/edit/persistentvolume/index.vue

```diff
@@ -196,12 +226,12 @@ export default {
       :namespaced="false"
     />
 
-    <InfoBox v-if="value.claim">
+    <InfoBox v-if="currentClaim">
       <div class="row">
         <div class="col span-6 text-center">
           <label class="text-muted">Persistent Volume Claim:</label>&nbsp;
-          <n-link :to="value.claim.detailLocation">
-            {{ value.claim.namespacedName }}
+          <n-link :to="currentClaim.detailLocation">
```

Comment:
The resources returned are raw json and not instances of our models... so this returns undefined and causes a white page error

Reply:
this is caused because we are bypassing the model... we don't use classify for the /request so I have to find another way.
Another way is using `create` action on `actions/dashboard-store`. Even better would be a `createAll` action... Targeted by type, preferably.  (note to myself)

Reply:
I am still having problems using the `cluster/create` action to classify the data, so for the sake of time, I don't think worth investing more hours into this right now. I then opted to update the needed props rather than using a model.

Reply:
This will work, but you would need to use `for` loops instead of `forEach`

```
        const types = Object.keys(hash);

        for (let i = 0; i < types.length; i++) {
          const type = types[i];

          const status = hash[type].status;
          // if it's namespaced, we get the data on 'items' prop, for non-namespaced it's  'data' prop...
          const requestData = hash[type].value.items || hash[type].value.data || hash[type].value;

          if (status === 'fulfilled' && resourceData.data[type] && resourceData.data[type].applyTo?.length) {
            for (let y = 0; y < resourceData.data[type].applyTo.length; y++) {
              const apply = resourceData.data[type].applyTo[y];
              const resources = await this.$store.dispatch('cluster/createMany', requestData);

              if (apply.parsingFunc) {
                this[apply.var] = apply.parsingFunc(requestData);
              } else {
                this[apply.var] = requestData;
              }
            }
          } else if (status === 'rejected') {
            console.error(`Resource Manager - secondary data request for type ${ type } has failed`, status.error); // eslint-disable-line no-console
          }
        }
```

```
  createMany(ctx, data) {
    return data.map(d => classify(ctx, d));
  },
```

Reply:
(we would still want to gate this functionality on some kind of `classify` boolean on `resourceData.data[type]`)